love/src/components/MainTel/M1M3TS/Selector/Selector.jsx

```diff
@@ -0,0 +1,413 @@
+        viewBox={`0 0 ${this.state.width} ${this.state.width}`}
+        onMouseEnter={this.disableScroll}
+        onMouseLeave={this.enableScroll}
+      >
+        {this.getBackground()}
+        {this.getScatter(
+          scale,
+          margin,
+          zoomLevel,
+          showFcuIDs,
+          showDifferentialTemp,
+          showWarning,
+          sensorSelect,
+          selectedSensor,
+        )}
+        {this.getAxis(margin, sensorSelect)}
+      </svg>
+    );
+  }
+
+  getScatter(scale, margin, zoomLevel, showFcuIDs, showDifferentialTemp, showWarning, sensorSelect, selectedSensor) {
+    return (
+      <g id="scatter" className={styles.scatter}>
+        {this.state.sensors.map((act) => {
+          return (
+            <g key={act.id} className={styles.sensor} onClick={() => sensorSelect(act.id)}>
+              <circle
+                cx={(act.position[0] + this.state.xRadius) * scale + margin}
+                cy={(act.position[1] + this.state.yRadius) * scale + margin}
+                key={act.id}
+                fill={
+                  showDifferentialTemp
+                    ? this.getSensor(act.id)?.colorDifferentialTemperature ?? 'gray'
+                    : this.getSensor(act.id)?.colorAbsoluteTemperature ?? 'gray'
+                }
+                stroke={selectedSensor === act.id ? this.strokeSensorSelected(act.id) : 'none'}
+                strokeWidth={act.id === selectedSensor ? 6 : 4}
+                r={(this.state.maxRadius * scale) / 16}
+                pointerEvents="all"
+              />
+              <text
+                x={(act.position[0] + this.state.xRadius) * scale + margin}
+                y={(act.position[1] + this.state.yRadius) * scale + margin}
+                textAnchor="middle"
+                alignmentBaseline="middle"
+                className={selectedSensor === act.id || (zoomLevel > 1 && showFcuIDs) ? '' : styles.hidden}
+                pointerEvents="none"
+              >
+                {act.id}
+              </text>
+              {showWarning && this.getSensor(act.id)?.warning ? (
+                <g
+                  transform-origin={`0% 0%`}
+                  transform={`translate(${(act.position[0] + this.state.xRadius) * scale + margin} ${
+                    (act.position[1] + this.state.yRadius) * scale + margin
+                  })`}
+                >
+                  {this.getWarningIcon(act.id)}
+                </g>
+              ) : (
+                <></>
+              )}
+            </g>
+          );
+        })}
+      </g>
+    );
+  }
+
+  getBackground() {
+    return (
+      <>
+        <circle
+          id="background-circle"
+          className={this.state.sensors.length > 0 ? styles.circleOverlay : styles.circleOverlayDisabled}
+          cx={this.state.width / 2}
+          cy={this.state.width / 2}
+          key={'background'}
+          r={this.state.width / 2}
+        />
+
+        {/* <circle
```

Comment:
Will this element be used? If not, maybe it should be removed.

Reply:
You are correct, removed on the next commit. Thank!